backend/app/api/endpoints/projects.py

```
@@ -0,0 +1,212 @@
+    List all projects for the current user.
+    Returns projects with optional task lists.
+    """
+    return project_service.list_projects(
+        db=db, user_id=current_user.id, include_tasks=include_tasks
+    )
+
+
+@router.post("", response_model=ProjectResponse, status_code=status.HTTP_201_CREATED)
+def create_project_endpoint(
+    project_create: ProjectCreate,
+    current_user: User = Depends(get_current_user),
```

Ruff (0.14.11)

Do not perform function call `Depends` in argument defaults; instead, perform the call within the function, or read the default from a module-level singleton variable

(B008)